App.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
Can we remove these:

```suggestion
```

Reply:
actually, I could not fix typescript-related issues in these files.
may be its related to the @types/react version issue.